cvat-canvas3d/src/typescript/canvas3dController.ts

```diff
@@ -3,8 +3,10 @@
 // SPDX-License-Identifier: MIT
 
 import { Canvas3dModel, Mode } from './canvas3dModel';
+import { DrawData } from '../../../cvat-canvas/src/typescript/canvasModel';
```

Comment:
You have ``DrawData`` in ``cvat-canvas3d/src/typescript/canvas3dModel.ts``. 
Probably this is wrong import.

Reply:
Resolved